map/map_loader/src/pointcloud_map_loader/utils.cpp

```diff
@@ -64,6 +64,7 @@ bool sphereAndBoxOverlapExists(
 bool isGridWithinQueriedArea(
   const autoware_map_msgs::msg::AreaInfo area, const PCDFileMetadata metadata)
 {
+  if (area.type == autoware_map_msgs::msg::AreaInfo::ALL_AREA) return true;
   // Currently, the area load only supports spherical area
```

Comment:
Can you also add 
```
else if (area.type == autoware_map_msgs::msg::AreaInfo::CYLINDER) {
  // code here
}
```

Reply:
Add conditional branch for CYLINDER in https://github.com/autowarefoundation/autoware.universe/pull/3162/commits/ae4e410edb2f66f34d20cd4d07a47e8d5d8b9124